.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: postgres:9.4
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    env:
+      RAILS_ENV: test
+      CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.2
```

Comment:
The `ruby-version` field is not needed with a `.ruby-version` file. https://github.com/ruby/setup-ruby#supported-version-syntax

I'd recommend to only keep the `.ruby-version` file, this way if we need to upgrade the Ruby version (which I also recommend 😜) there is only one file to change

Reply:
[Change commited here](https://github.com/rootstrap/rails_api_base/pull/345/commits/e590faa831b7880d9440bfb036ee69be3deb4552)